pagopa/io-app
# chore(IDPay): [IODPAY-195] New IDPay bonus card in wallet

### ⚠️ Depends on https://github.com/pagopa/io-app/pull/4575

## Short description
This PR adds the new design for the IDPay bonus card in wallet.

https://user-images.githubusercontent.com/6160324/235421707-1fc9d812-0c6a-4faa-b215-6959b83b41ca.mp4

## List of changes proposed in this pull request
- Added new card design
- Added assets 
- Updated API specs for `logoURL` and `organizationName` in `InitiativeDTO`

## How to test
With `io-dev-server`, navigate to the wallet section of the app and check if the new card is correctly displayed
